locales/de/calendar.json

```diff
@@ -0,0 +1,48 @@
+{
+  "date": "Datum",
+  "time": "Uhrzeit",
+  "event": "Formel 1 Grand Prix Veranstaltungen",
+  "notice": {
+    "text": "Die ersten acht Rennen des revidierten Kalenders unten. Times are TBC along with additional races.",
+    "link": "Mehr lesen"
+  },
+  "badges": {
+    "tbc": "Unbestätigt",
+    "tickets": "Karten",
+    "canceled": "Abgesagt",
+    "next": "NÄCHSTER"
+  },
+  "schedule": {
+    "fp1": "Freies Training 1",
+    "fp2": "Freies Training 2",
+    "fp3": "Freies Training 3",
+    "qualifying": "Qualifikation",
+    "race": "Rennen"
+  },
+  "races": {
+    "austrian-grand-prix": "Großer Preis von Österreich",
+    "steiermark-grand-prix": "Großer Preis der Steiermark",
+    "hungary-grand-prix": "Großer Preis von Ungarn",
+    "british-grand-prix": "Großer Preis von Großbritannien",
+    "anniversary-british-grand-prix": "70er Geburtstags - Grand Prix (Großbritannien)",
+    "spanish-grand-prix": "Großer Preis von Spanien",
+    "belgian-grand-prix": "Großer Preis von Belgien",
+    "italian-grand-prix": "Großer Preis von Italien",
+    "australian-grand-prix": "Großer Preis von Australien",
+    "bahrain-grand-prix": "Großer Preis von Bahrain",
+    "vietnamese-grand-prix": "Großer Preis von Vietnam",
+    "china-grand-prix": "Großer Preis von China",
+    "dutch-grand-prix": "Großer Preis der Niederlande",
+    "monoco-grand-prix": "Großer Preis von Monaco",
+    "azerbaijan-grand-prix": "Großer Preis von Aserbaidschan",
+    "canadian-grand-prix": "Großer Preis von Kanada",
+    "french-grand-prix": "Großer Preis von Frankreich",
+    "singapore-grand-prix": "Großer Preis von Singapur",
+    "russian-grand-prix": "Großer Preis von Russland",
+    "japanese-grand-prix": "Großer Preis von Japan",
+    "us-grand-prix": "Großer Preis der USA",
+    "mexico-grand-prix": "Großer Preis von Mexiko-Stadt",
```

Comment:
Why Mexiko-Stadt when always just using the country?

Reply:
In the English Version it was also the Mexico City Grand Prix (They changed it for this year). Probably this should be two complete separate events for backwards compatibility. Or what do you mean @ay8s ?